tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -988,9 +988,19 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
                               default=())
     # TODO: isn't it a weights precision type
     common_group.add_argument('--data_type',
-                              help=mo_convert_params['data_type'].description,
+                              help='[DEPRECATED] Data type for model weights and biases. '
+                                   'If original model is in FP32 and --data_type=FP16 is specified, '
+                                   'model weights and biases are compressed to FP16, except for const values which '
+                                   'do not fit in FP16 range. All intermediate data is kept in original precision.',
```

Comment:
why do we put text here from `ParamDescription`? I would suggest to make changes there and leave this line as is

Reply:
Because there was requirement to not add deprecated parameters to convert_model(). More details in comment below.